app/scripts/templates/settings/emails.mustache

```diff
@@ -0,0 +1,60 @@
+<div id="emails" class="settings-unit">
+    <div class="settings-unit-stub">
+        <header class="settings-unit-summary">
+            <h2 class="settings-unit-title">{{#t}}Secondary email{{/t}}</h2>
+        </header>
+        <button class="settings-button secondary settings-unit-toggle hidden" data-href="/settings/emails">
+            {{#hasSecondaryEmail}}{{#t}}Change…{{/t}}{{/hasSecondaryEmail}}
+            {{^hasSecondaryEmail}}{{#t}}Add…{{/t}}{{/hasSecondaryEmail}}
+        </button>
+    </div>
+
+    <div class="settings-unit-details">
+        <div class="error"></div>
+
+        <form novalidate>
+            {{#hasSecondaryEmail}}
+                <p>
+                    {{#t}}Current secondary email associated with account.{{/t}}
+                </p>
+                <ul class="email-list button-row">
+                    {{#emails}}
+                        {{^isPrimary}}
+                            <li class="email-address">
+                                <div class="address">{{email}}</div>
+                                <div class="details">
+                                    {{#isVerified}}
+                                        verified
+                                    {{/isVerified}}
+                                    {{^isVerified}}
+                                        not verified
+                                    {{/isVerified}}
+                                </div>
+                                <button class="settings-button warning email-disconnect" data-id="{{email}}">
+                                    {{#t}}Remove...{{/t}}
+                                </button>
+                            </li>
+                        {{/isPrimary}}
+                    {{/emails}}
+                </ul>
+            {{/hasSecondaryEmail}}
+
+            {{^hasSecondaryEmail}}
+                <div class="input-row">
+                    <p>
+                        {{#t}}
+                            A secondary email is an additional address for receiving security
```

Comment:
Can you pull this line to start just after the closing `}` and all on a single line, otherwise there will be extra space in the translations.